pkg/virt-operator/webhooks/kubevirt-update-admitter.go

```diff
@@ -61,6 +65,14 @@ func (admitter *KubeVirtUpdateAdmitter) Admit(ar *admissionv1.AdmissionReview) *
 	results = append(results, validateCustomizeComponents(newKV.Spec.CustomizeComponents)...)
 	results = append(results, validateCertificates(newKV.Spec.CertificateRotationStrategy.SelfSigned)...)
 
+	if newKV.Spec.Infra != nil && newKV.Spec.Infra.NodePlacement != nil {
```

Comment:
What about comparing the `Infra` and `Workloads` from old to new KV? Couldn't it save some network ops and reduce avg. latency?

Reply:
@xpivarc You are correct! thanks. Changed the code